src/containers/edm/propertytypes/PropertyTypeDetailsContainer.js

```diff
@@ -78,15 +178,102 @@ class PropertyTypeDetailsContainer extends React.Component<Props> {
     );
   }
 
-  render() {
+  renderPropertyTypePiiSection = () => {
 
     const { propertyType } = this.props;
     if (!propertyType || propertyType.isEmpty()) {
       return null;
     }
 
-    const ptPII :boolean = propertyType.get('pii', false);
-    const piiAsString :string = ptPII === true ? 'true' : 'false';
+    const piiValue :boolean = propertyType.get('pii', false);
+
+    return (
+      <section>
+        <h2>PII</h2>
+        <RadiosRowWrapper>
+          <label htmlFor={PII_YES_RADIO_ID}>
+            <input
```

Comment:
you could always use the `Label` component from LUK 🤷🏽‍♀️And sounds like also the radio button very soon :)

Reply:
Gonna LUK-ify the whole app soon.

Reply:
Never tried using `Label` from LUK for this application. `Radio` and `Checkbox` come wrapped with their own label. However, you should use Label for describing a group of Radios or Checkboxes.